test-app/readme.md

```diff
@@ -0,0 +1,48 @@
+# Createjs Accessibility Module Test App and Reference Implementation
+
+## Description
+The Createjs Accessibility Module (abbreviated CAM, located at https://github.com/CurriculumAssociates/createjs-accessibility) allows for annotating DisplayObjects with accessibility information which is used to translate the DisplayObjects to the DOM to help a web page's canvas content to meet WCAG 2.1.  This test app allows for testing any changes to that module in browsers that may or may not have an Assistive Technology (AT) in use.  Also, some of the ARIA roles require a particular nesting structure to function correctly (e.g. menubar) and this test app also servers as a reference implementation of doing that with CAM.
+
+## Setup
+Since this is a sample app and not a library, the first step is to clone the repo:
+```
+git clone https://github.com/CurriculumAssociates/createjs-accessibility-tester.git
```

Comment:
Looks like we either want to remove to update this accordingly 🙂

Reply:
Update: Moved the info into the main readme and updated with instructions for running + GitHub page URL for after everything's merged